lib/src/anchor.dart

```diff
@@ -3,19 +3,28 @@ import 'package:flutter/widgets.dart';
 import 'package:flutter_html/src/styled_element.dart';
 
 class AnchorKey extends GlobalKey {
+  static final Set<AnchorKey> _registry = <AnchorKey>{};
```

Comment:
Just curious, how does this work correctly? Every time you make a new `AnchorKey`, wouldn't this variable be cleared? So every time `.of` is called, it finds no existing keys in `_registry`? 

I thought that new class instances never had variables persist between them, or maybe a new `AnchorKey` class is never created...

Reply:
Duplication of GlobalKeys is not problematic for creating them (in fact, the constructor is empty) but to have multiple of them in the widget tree. `AnchorKey`s are `GlobalKey`s and so instantiating them, even with the same parentKey and id, work fine. It is not until you actually add them as key to some widgets that the issue occurs.

Of course my solution relies on that `forId` is only used for lookups while `of` is only used for 'creation with the intention of using as widget key'. I considered renaming the function but actually couldn't come up with a better name.